.circleci/config.yml

```diff
@@ -1,5 +1,8 @@
 version: 2.1
 
+orbs:
+  codecov: codecov/codecov@3
```

Comment:
What repo does this point to? I cannot find https://github.com/codecov/codecov , only only https://github.com/codecov/codecov-circleci-orb

Reply:
Oh nvm, I see it https://circleci.com/developer/orbs/orb/codecov/codecov

Reply:
I guess another thing to check is whether this is available for our tier or you have to pay extra... 🤷